proof-general-pkg.el

```diff
@@ -0,0 +1,2 @@
+(define-package "proof-general" "4.4.1~pre" "A generic front-end for proof assistants (interactive theorem provers)"
+  '((emacs "24.4")))
```

Comment:
Thanks.  I think that's probably true, but I haven't tested it there, and I don't think we're regularly doing that, so I'd rather be conservative.  Do you know many users of 24.3?